components/web/ejabberd/Makefile

```diff
@@ -21,60 +21,77 @@
 
 #
 # Copyright (c) 2011, 2015, Oracle and/or its affiliates. All rights reserved.
+# Copyright 2017 Andreas Grueninger, Grueninger GmbH, (grueni). All rights reserved. 
 #
 
 include ../../../make-rules/shared-macros.mk
 
-# for ginstall and tail -n
-PATH=/usr/gnu/bin:/usr/bin
-
-COMPONENT_NAME=		ejabberd
-COMPONENT_VERSION=	2.1.13
+COMPONENT_NAME= ejabberd
+COMPONENT_FMRI= web/server/$(COMPONENT_NAME)
+COMPONENT_VERSION= 17.11
 COMPONENT_PROJECT_URL=	http://www.ejabberd.im/
-COMPONENT_SRC=		$(COMPONENT_NAME)-$(COMPONENT_VERSION)
-COMPONENT_ARCHIVE=	$(COMPONENT_SRC).tgz
-COMPONENT_ARCHIVE_HASH=	\
-    sha256:a80d1c72e435da31758261cb0c04fe750ac71fc3e33a3b606123efd3b3abc86d
-COMPONENT_ARCHIVE_URL=	http://www.process-one.net/downloads/ejabberd/$(COMPONENT_VERSION)/$(COMPONENT_ARCHIVE)
-COMPONENT_BUGDB=	service/ejabberd
-
-include ../../../make-rules/prep.mk
-
-SRC_OFFSET =	src
-# Now we override paths to .built
-BUILD_DIR_32 = $(BUILD_DIR)/$(MACH32)/$(SRC_OFFSET)
-
-COMPONENT_PREP_ACTION = \
-       (cd $(@D)/src ; PATH=$(PATH) $(AUTORECONF) -fiv)
-
-
-include $(WS_TOP)/make-rules/configure.mk
-include $(WS_TOP)/make-rules/ips.mk
+COMPONENT_SRC= $(COMPONENT_NAME)-$(COMPONENT_VERSION)
+COMPONENT_ARCHIVE= $(COMPONENT_SRC).tgz
+COMPONENT_ARCHIVE_HASH= sha256:bd16b33a2f1345e5847fd05b70a5fa992181bac756cd6d5d621de9c18cb76801
+COMPONENT_ARCHIVE_URL= https://www.process-one.net/downloads/ejabberd/$(COMPONENT_VERSION)/$(COMPONENT_ARCHIVE)
+COMPONENT_BUGDB= service/ejabberd
+COMPONENT_SUMMARY= ejabberd - Jabber/XMPP instant messaging server
+COMPONENT_DESCRIPTION= the ejabberd Jabber/XMPP instant messaging server
+COMPONENT_CLASSIFICATION= Web Services/Communications
+COMPONENT_LICENSE= GPL-2.0
+COMPONENT_LICENSE_FILE= $(COMPONENT_NAME).license
+
+include $(WS_MAKE_RULES)/prep.mk
+include $(WS_MAKE_RULES)/configure.mk
+include $(WS_MAKE_RULES)/ips.mk
+
+CC=gcc -m64 -I/usr/include/gd2
+CPP=gcc -m64 -E -I/usr/include/gd2
+CXX=g++ -m64 -I/usr/include/gd2
+LDFLAGS += -L/usr/lib/libjpeg8-turbo/lib/amd64 -R/usr/lib/libjpeg8-turbo/lib/amd64 -lnsl -lsocket
```

Comment:
s/amd64/$(MACH64)/

Reply:
You should use the predefined macros instead, for reference:
    `CFLAGS   += $(JPEG_CPPFLAGS) $(JPEG_CFLAGS)`
    `CXXFLAGS += $(JPEG_CPPFLAGS) $(JPEG_CXXFLAGS)`
    `LDFLAGS  += $(JPEG_LDFLAGS)`